content/faq/31_what-do-i-do-if-i-get-error-while-coding/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: what do i do if i get error while coding?
```

Comment:
this should be:
`what do I do if I get an error in a public library which I am using?`

Reply:
Done.